.github/workflows/coverage.yml

```diff
@@ -13,7 +13,8 @@ jobs:
       - uses: actions/checkout@v1
       - uses: actions-rs/toolchain@v1
         with:
-          toolchain: nightly-2022-03-22
+          #toolchain: nightly-2022-03-22
+          toolchain: nightly
```

Comment:
I don't know if there's been a discussion about this, but this is probably a bad idea. Nightly sometimes have bugs and things aren't stable. We should pick a date and stick to it. If there's a feature that doesnt' work with an old nightly, just switch to last night's nightly.

Reply:
Unfortunately, some features wasn't stable on `nightly-2022-03-22`, for example `bool_to_option` feature that provide `then_some` function.  

I understand risks, but looks the current nightly works fine, also the current stable works fine too. Btw, the stable version have more features than `nightly-2022-03-22`. So, I see here a couple of solutions:
1. We can use the latest stable, that might confront with previous discussions. But `nightly-2022-03-22` more outdated than the current stable. 
2. Stick with the current nightly. 

I agree with both variants, because a nightly build that released almost 6 months ago, it's quite outdated.

Reply:
I think you misunderstood what I'm saying. I'm not saying to stick to 22-03-2022. I'm saying use the one from yesterday, 16-08-2022, instead of keeping it unbounded with just `nightly`.

If it works with stable, that's even better. I don't really know why we're using nightly, but obviously I imagine there's something not ready for coverage on stable.

Reply:
I understood you, no worries. Ok, let's leave yesterday nightly version, and when we have free hands to inspect is coverage ready for the stable, then change it on in another branch. 

p.s. As far as I remember, we started to use nightly due to substrate, it doesn't build on stable. And then we just kept it as legacy.